<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to quiz</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>390</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0248052, y1:0.011, x2:0.955039, y2:0.926, stop:0 rgba(209, 104, 165, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>NEW USER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Log_in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
